ares.h

```diff
@@ -265,6 +272,8 @@ struct ares_options {
   struct apattern *sortlist;
   int nsort;
   int ednspsz;
+  ares_debug_callback debug_cb;
+  void *debug_cb_data;
```

Comment:
According to the comment above this structure, I think the preferred way to do options now is to have a separate `ares_set_debug_callback(channel, cb, data)` entrypoint.